### ISSUE TITLE
Add in README file - Authorization problem despite being assigned as Admin in aws

### ISSUE THREAD

**Issue by nataliey16** [Code-the-Change-YYC/hackathon-management-tool#87]
Add a section about clearing cookies and/or using incognito to the README file if someone is initially having trouble performing Admin tasks. When I first updated my user role, I did not have the accessibility I was supposed to as admin. This was only solved when going into incognito mode. But I eventually gained full access as admin but I think there is a update delay.

**Comment by ideen1** [comment]
Good catch, did you happen to log out/log back in after changing the role for the user? The role update will only take effect after a login since that is when the ID/access tokens are updated
